pkg/proxydetect/proxydetect_nonwindows.go

```diff
@@ -0,0 +1,13 @@
+// +build !windows
+
+package proxydetect
+
+import "net/url"
+
+// todo: darwin proxy detection
```

Comment:
Do we really need this TODO in code? We have a lot if similar features that are implemented only for one GOOS, and they don't have TODO comment.

Reply:
agree, removed